shared/js/textChatView.jsx

```diff
@@ -87,21 +90,32 @@ loop.shared.views.chat = (function(mozL10n) {
         );
       }
 
+      // Only render messages after they've come back from the server to avoid
+      // dups in a simple way for early product standup.
+      //  XXX akita Probably not the UX we want; should talk to UX folks.
+      if (this.props.type === CHAT_MESSAGE_TYPES.SENT) {
+        return null;
```

Comment:
Yes, it does.  

I've filed bug 1279054 for unregressing the speech bubbles and put it in the XXX comment.